.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: meta-introspector/checkout@v2
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)